packages/base/src/MediaRange.ts

```diff
@@ -1,8 +1,36 @@
-const querySets = new Map<string, RangeSet>();
+type Range = Map<string, Array<number>>;
 
-type RangeSet = {
-	borders: Array<number>,
-	names: Array<string>,
+const mediaRanges = new Map<string, Range>();
+
+const DEAFULT_RANGE_SET: Range = new Map<string, Array<number>>();
+DEAFULT_RANGE_SET.set("S", [0, 599]);
+DEAFULT_RANGE_SET.set("M", [600, 1023]);
+DEAFULT_RANGE_SET.set("L", [1024, 1439]);
+DEAFULT_RANGE_SET.set("XL", [1440, 2560]);
```

Comment:
For "XL" breakpoint, you better use "Infinity", instead of "2560" to ensure the code works as before. Later, we can introduce a new breakpoint (2XL) and then add the "2560" breakpoint as here.